sdk/eventhub/service.projects

```diff
@@ -0,0 +1,38 @@
+<!--
+  This file allows overrides and extensions to the build configuration defined
+  by the central engineering system, and is considered during pipeline builds for
+  CI, test runs, and package release.
+
+  Based on the SDKType variable (msbuild property or env variable) this will filter
+  the set of projects to to match what type of SDK we are interested in building.
+
+  SDKType values
+   - empty or client will build the new track 2 client/data libraries
```

Comment:
(extreme) nit:  It may be more clear if we quote the magic strings.    For example:

```suggestion
   - empty or "client" will build the new track 2 client/data libraries
```